### PR TITLE
[To rel/1.0] Cherry pick cluster name related commits and SQL SHOW VARIABLES

### PR DESCRIPTION
**Pull request by CRZbulabula**

### CODE REVIEW

File: docs/UserGuide/Cluster/Cluster-Maintenance.md [hunk 1]
```diff
@@ -21,18 +21,18 @@
...
+SHOW VARIABLES
 ```
 
 Eg:
 ```
-IoTDB> show cluster parameters
+IoTDB> show variables
 +----------------------------------+-----------------------------------------------------------------+
-|                 ClusterParameters|                                                            Value|
+|                         Variables|                                                            Value|
```
Comment (Beyyes) [inline docs/UserGuide/Cluster/Cluster-Maintenance.md:35]:
```suggestion
|                         Variable|                                                            Value|
```

File: docs/zh/UserGuide/Cluster/Cluster-Maintenance.md [hunk 2]
```diff
@@ -25,14 +25,14 @@
...
+SHOW VARIABLES
 ```
 
 示例：
 ```
-IoTDB> show cluster parameters
+IoTDB> show variables
 +----------------------------------+-----------------------------------------------------------------+
-|                 ClusterParameters|                                                            Value|
+|                         Variables|                                                            Value|
```
Comment (Beyyes) [inline docs/zh/UserGuide/Cluster/Cluster-Maintenance.md:35]:
```suggestion
|                         Variable|                                                            Value|
```

Comment (CRZbulabula) [reply docs/zh/UserGuide/Cluster/Cluster-Maintenance.md]: Fixed~

File: docs/UserGuide/Cluster/Cluster-Maintenance.md (hunk 1 above)
Comment (CRZbulabula) [reply docs/UserGuide/Cluster/Cluster-Maintenance.md]: Fixed~

**Review by Beyyes — APPROVED** [review]
LGTM

**Pull request merged**